### DISCUSSION TITLE
Usage

### DISCUSSION THREAD

**Discussion by astalavista321** [paveloom-f/lbry-desktop#7] [Q&A]
Might sounds like a silly question but how does one apply this fork? Is installing from the release section enough or some of the files from the zip have to be applied somehow to test the functionality of the filter removal?

**Comment by paveloom** [comment]
@astalavista321 Downloading a binary for your system from the `Releases` page and launching it is enough. It will install and launch the program for you, so you don't need to have an official version installed. To test if it works, you may check out links from the notices in the LBRY's [DMCA repository](https://github.com/lbryio/dmca).

**Reply by astalavista321** [reply]
Thank you for the detailed reply. I tried the DCMA links and they open in a new tab on the browser I'm using. How would you open a test DCMA link using the released installation?

**Reply by paveloom** [reply]
@astalavista321 Copy the name of the file (a combination of symbols after `odysee.com/` or `lbry.tv/`) from a file's URL and paste it in the search box inside the application. Press <kbd>Enter</kbd> to search. The file should be the first item on the results page.
